.papr.yml

```diff
@@ -19,9 +30,10 @@ packages:
   - openssl-devel
   - redhat-rpm-config
 
-context: 'fedora/25/atomic | origin/v3.6.0-alpha.1'
+context: 'fedora/25/atomic | origin/v3.6.0-alpha.2'
```

Comment:
How is this `context` key used?

Could we somehow remove the duplication of this value here and in OPENSHIFT_IMAGE_TAG?